sklearn/tests/test_multiclass.py

```diff
@@ -470,16 +482,28 @@ def test_ovo_partial_fit_predict():
     assert_almost_equal(pred1, pred2)
 
     # Test when mini-batches don't have all target classes
-    ovo1 = OneVsOneClassifier(MultinomialNB())
-    ovo1.partial_fit(iris.data[:60], iris.target[:60], np.unique(iris.target))
-    ovo1.partial_fit(iris.data[60:], iris.target[60:])
```

Comment:
Why did you remove this? It looks like this is as correct as well. The first pass just has classes `[0, 1]` and the second pass does not have class 0